### ISSUE TITLE
Make field order generated by flask spec deterministic

### ISSUE THREAD

**Issue by lorenyu** [apiflask/apiflask#385]
The `flask spec` command should output fields in a deterministic order. Currently when running `flask spec --format yaml > ./openapi.yml` multiple times in a row, the file changes every time.

**Comment by greyli** [comment]
Thanks for reporting this. This should be fixed with #387.

I will make a new release no later than 31 December. Before that, you can keep the schema in order by adding a base schema for all your schemas:

```
from apiflask import Schema

class OrderedSchema(Schema):
    class Meta:
        ordered = True


class PetSchema(OrderedSchema):
    pass
```

**Comment by lorenyu** [comment]
Awesome, thanks! And thanks for the workaround too! 👍
